charts/fdb-operator/templates/rbac/rbac_role_binding.yaml

```diff
@@ -23,3 +23,17 @@ subjects:
   {{- if .Values.globalMode.enabled }}
   namespace: {{ .Release.Namespace }}
   {{- end }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
The change itself look good, but I think it makes sense to have this optional.

e.g. by surrounding the new `ClusterRoleBinding` and `ClusterRole` with:

 ```yaml
{{- if .Values.nodeReadClusterRole }}
---
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRoleBinding
metadata:
  name: {{ include "fdb-operator.fullname" . }}-clusterrolebinding
  labels:
    {{- include "fdb-operator.labels" . | nindent 4 }}
roleRef:
  apiGroup: rbac.authorization.k8s.io
  kind: ClusterRole
  name: {{ include "fdb-operator.fullname" . }}-clusterrole
subjects:
- kind: ServiceAccount
  name: {{ include "fdb-operator.serviceAccountName" . }}
{{- end }}
```

What do you think?